src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java

```diff
@@ -32,7 +32,16 @@ public void acquire() {
     public void flapOut() {
         crabArmMotors.set(-1 * RobotMap.FLAP_OUT_SPEED);
     }
-
+    
+    public static void flapPush() {
```

Comment:
What is the difference between `flapPush()` and `flapCoast()`?
If the differentiation is necessary, please comment to explain.
If not, remove the redundant code.